hibernate-core/src/test/java/org/hibernate/test/legacy/CustomPersister.java

```diff
@@ -309,9 +309,10 @@ public Object load(
 		Serializable id,
 		Object optionalObject,
 		LockOptions lockOptions,
-		SharedSessionContractImplementor session
+		SharedSessionContractImplementor session,
+		Boolean readOnly
```

Comment:
IMO this class should compile without changes

Reply:
Reverted back. Thanks.